src/fragments/lib-v1/auth/ios/getting_started/50_configureBackend.mdx

```diff
@@ -0,0 +1,200 @@
+<BlockSwitcher>
+<Block name="Amplify CLI (Create)">
+
+> Prerequisites: [Install and configure](/gen1/prev/[platform]/start/project-setup/prerequisites/#install-and-configure-the-amplify-cli) the Amplify CLI in addition to the Amplify libraries and [necessary dependencies](/gen1/prev/[platform]/build-a-backend/auth/set-up-auth/#install-amplify-libraries).
+
+To start provisioning auth resources in the backend, go to your project directory and **execute the command**:
+
+```bash
```

Comment:
can we add `title="Terminal" showLineNumbers={false}` to each of these bash snippets where we're showing a command that should be executed in the terminal?